index.html

```diff
@@ -28,8 +28,8 @@ <h3>We are one of the most successful and inventive FIRST Robotics teams in the
 <main>
     <div>
         <img src="/assets/img/team.jpg">
-        <p>We are team 1418 Vae Victis, a <a href="http://firstinspires.org/robotics/frc">FIRST Robotics Competition (FRC)</a> team from George Mason High School in Falls Church, VA. Since our founding in 2004, our team has grown in every way, and has maintained a high place in competition and in the community. To do this, we maintain an active and engaged student body of around fifteen students who work with the team on and off season.</p>
-        <p>We are proud to have experienced success in competition. We have made it to the FIRST World Championship in St. Louis, Missouri for the past two years, and plan to continue our streak. Outside of competition, we pride ourselves on involvement in our community. We have hosted free summer camps to educate elementary school students on basic LEGO Robotics, and volunteer to help run FIRST Lego League competitions every year. We've even helped to start robotics teams in rural Ecuador and help educate students living there. More information about our outreach efforts can be found <a href="outreach">here</a>.</p>
+        <p>We are team 1418 Vae Victis, a <a href="http://firstinspires.org/robotics/frc">FIRST Robotics Competition (FRC)</a> team from George Mason High School in Falls Church, VA. Since our founding in 2004, our team has grown in every way, and has maintained a high place in competition and in the community. To do this, we maintain an active and engaged student body of around thirty students who work with the team during both the on and off season.</p>
```

Comment:
I think it's more like 20 or 25 if we look at who's actually engaged.

The wording in the final sentence is intentional.